luatest/server.lua

```diff
@@ -480,6 +480,17 @@ function Server:eval(...)
     if self.net_box == nil then
         error('net_box is not connected', 2)
     end
+    self.net_box:eval([[
```

Comment:
This function may look like this:

```lua
function Server:eval(code, ...)
    if self.net_box == nil then
        error('net_box is not connected', 2)
    end
    local preamble = [[
        if not rawget(_G, 't') then
            local is_ok, _t = pcall(require, 'luatest')
            if is_ok then
                rawset(_G, 't', _t)
            else
                require('log').warn('LUA_PATH is unset or incorrect, ' .. _t)
            end
        end
    ]]
    return self.net_box:eval(preamble .. code, ...)
end
```

In such a way, we will have only one `self.net_box:eval` call.

Reply:
Done. Thanks!